src/PKSim.Core/ApplicationStartup.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Globalization;
+using System.Reflection;
+
+namespace PKSim.Core;
+
+public class ApplicationStartup
+{
+   protected static void RedirectAssembly(string shortName, Version targetVersion, string publicKeyToken)
+   {
+      Assembly Handler(object sender, ResolveEventArgs args)
+      {
+         var requestedAssembly = new AssemblyName(args.Name);
+         if (requestedAssembly.Name != shortName) return null;
```

Comment:
`return null` instead of just `return `in a `void`-function - somehow strange for me

Reply:
Oh, I guess you mean because of this closure.